disable-comments.php

```diff
@@ -235,7 +235,11 @@ public function filter_wp_headers( $headers ) {
 	 */
 	public function filter_query() {
 		if( is_comment_feed() ) {
-			wp_die( __( 'Comments are closed.' ), '', array( 'response' => 403 ) );
+			$refere = isset($_SERVER['HTTP_REFERER'])? $_SERVER['HTTP_REFERER']: '';
+			$location = !empty($refere)? htmlspecialchars($refere): home_url('/');
```

Comment:
Yes, this line is apparently not good.
But Wordpress error page is disadvantaged.
I will remove this line.
How about do you think redirect? stupid thing?